src/config-json-schemas/Schema.json

```diff
@@ -239,12 +266,46 @@
       ]
     },
 
+    "AfterAction": {
+      "type": "object",
+      "properties": {
+        "type": {
+          "type": "string",
+          "enum": [
+            "task",
+            "notifyAlgoQueueManager"
+          ]
+        }
+      },
+      "anyOf": [
+        { "$ref": "#/definitions/TaskAction" },
+        { "$ref": "#/definitions/NotifyAlgoQueueManagerAction" }
+      ]
+    },
+
+    "NotifyAlgoQueueManagerAction": {
```

Comment:
Would it make sense to indicate this boi is deprecated? There should be an example in this file. Just a warning in a `description` field seem to show up nicely when auto completing.

Last time I did not implement this as it is currently not used and should/can not be used. But this does match what the data service technically accepts. Happy to discuss